dogen/templates/template.jinja

```diff
@@ -102,6 +102,7 @@ RUN [ "bash", "-x", "/tmp/scripts/{{ script.package }}/{{ script.exec }}" ]
 {% if cct %}
 ADD cct /tmp/cct/
 ENV CCT_MODULES_PATH=/tmp/cct/
+USER {{ cct.user }}
```

Comment:
I don't think we have a previous user as such; each section needs to set the user for itself that it needs (e.g. scripts). Actually we don't seem to have a top-level user: key for setting the user as the last instruction that the CMD is run at? We probably should